curation-authors-backfill/index.ts

```diff
@@ -44,48 +45,14 @@ export async function handlerFn(event: SQSEvent): Promise<SQSBatchResponse> {
         ];
       }
 
-      // Wait a sec... don't barrage the api. We're just backfilling here.
+      // Wait a sec... don't barrage the API. We're just backfilling here.
+      await sleep(1000);
 
-      //await sleep(1000);
-
-      //const mutationResponse = await callUpdateMutation({
-      //  externalId,
-      //  authors,
-      //});
-
-      // TODO: do something with the response...
-
-      // copy / pasta code below - keeping for reference for now
-
-      //as json stringify could throw error in catch, which can cause entire batch failure
-      // fetching this value in try, and using them in catch.
-      // curatedRecId = message.curated_rec_id;
-      // resolvedUrl = message.resolved_url;
-      // resolvedId = message.resolved_id;
-      // imageUrl = message.image_src;
-      // const prospectData = await fetchProspectData(message.resolved_url);
-      // const corpusInput = hydrateCorpusInput(message, prospectData);
-      // // Wait a sec... don't barrage the api. We're just backfilling here.
-      // await sleep(1000);
-      // const importMutationResponse = await callImportMutation(corpusInput);
-      // const curatedItemRecord: CuratedItemRecord = {
-      //   curatedRecId: parseInt(message.curated_rec_id),
-      //   scheduledItemExternalId:
-      //     importMutationResponse?.data?.importApprovedCorpusItem.scheduledItem
-      //       .externalId,
-      //   approvedItemExternalId:
-      //     importMutationResponse?.data?.importApprovedCorpusItem.approvedItem
-      //       .externalId,
-      //   scheduledSurfaceGuid:
-      //     ScheduledSurfaceGuid[
-      //       importMutationResponse?.data?.importApprovedCorpusItem.scheduledItem
-      //         .scheduledSurfaceGuid
-      //     ],
-      //   lastUpdatedAt: new Date().getTime(),
-      // };
-      // console.log(`curatedItemRecord -> ${JSON.stringify(curatedItemRecord)}`);
-      //
-      // await insertCuratedItem(dbClient, curatedItemRecord);
+      // Run the `updateApprovedCorpusItemAuthors` mutation
+      await callUpdateMutation({
```

Comment:
just curious - do we want to do anything with this response? log it just in case?

feels like it would be nice to have some kind of logging as a first step to debugging in case something went goofy. maybe the data going into the mutation and the data returned?

Reply:
@kkyeboah do you have a code sample we can copy pasta?

Reply:
https://github.com/Pocket/curation-tools-data-sync/blob/44997ecfb5d8466202bc9c4e50915315c5fce026/curation-migration-backfill/externalCaller/importMutationCaller.ts#L25 we used `backOff` here

Reply:
ah, okay, okay. we already have this code copied in 😄